### PR TITLE
Add False Flags verbs under admin Secrets panel

### PR DESCRIPTION
**Pull request by ghost**
All those verbs have their own category, a typo has also been fixed
- Added the False Flags and Bait category, under +FUN flag
- Fake Biohzard Alert : Allows the admin to fire any biohazard level supported by the announcement system (1 to 7, 0 is random). If out of range input, cancel
- Fake Lifesigns Alert : Fake Alium/Spider alert message
- Fake Delta : Fake Delta Warning. Admin can choose to add Malfunction specific message, or leave it away for mild baiting. Yes, I know, it's dangerous for the AI
- Fake Booms : Sometimes copypasting "sound "[explosion]"" just won't do
- Fake Ion : Some people will go EXTREMELY far to know about that law
  the AI "got"
- Fake Meteor : The Large Wave Event variant. The meteors never comes, but the crew morale is impacted regardless
- Fake Carps : An odd one, but hey

Those are intended for three reasons :
- Metagame/Powergame baiting. Hear a Biohazard Alert five minutes into the round ? BLUB. Fake Ion and the AI is not stating an Ion Law ? BLOW EVERYTHING
- General utility. The Fake Booms command is nice if you want a lot of booms without actually making booms or copypasting the sound command
- General fun, because +FUN

### CODE REVIEW

**Comment by ririchiyo** [comment]
This is dumb, rework it all into one method.

**Comment by dylanstrategie** [comment]
How so ?

**Comment by Lokiamis** [comment]
>Fake Ion and the AI is not stating an Ion Law ? BLOW EVERYTHING

Who are these fun for? Ghosts?

**Comment by dylanstrategie** [comment]
The admins are responsible for the buttons they press

What is an instant supermatter cascade fun for ?

**Comment by IratePirate** [comment]
I'm alright with this. Anything that breaks up the metagame is inherently good. It should be like a shitter safari for people who blow borgs at the drop of a hat.

**Comment by Aurx** [comment]
Does the fake delta also start a fake timer?

**Comment by dylanstrategie** [comment]
No it doesn't. It would most likely cause a lot of problems. Furthermore Fake Delta means fake, Delta Alert is never set

**Comment by ririchiyo** [comment]
Dylan, when I mean make it all into one method I mean to make it so that you can completely customize the message, sound, and any other thing that might be possible. This way we can just have one method for it instead of having to have multiple verbs for this in the html window which doesn't make much sense since they will all do the same exact thing.

````
initFakeMessage(message, sound, args) {

}
````

This way you'd be able to have a message passed into it, have sound passed into it, and any other additional arguments you might think to add. There's no need multiple commands that all do almost the same thing. It would probably be better to rework the global announcement system into this as well so it's all under one thing.

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -2633,6 +2633,57 @@
...
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKET")
+				var/choice = input("Do you wish to include the Hostile Runtimes warning to have an authentic malfunction takeover alert ?") in list("Yes", "No")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Malfunction Takeover Alert (Hostile Runtimes alert [choice == "Yes" ? "included":"excluded"])")
+				if(choice == "Yes")
+					command_alert("Hostile runtimes detected in all station systems, please deactivate your AI to prevent possible damage to its morality core.", "Anomaly Alert")
+				world << "<font size=4 color='red'>Attention! Delta security level reached!</font>" //Don't ACTUALLY set station alert to Delta to avoid fucking shit up for real
+				world << "<font color='red'>[config.alert_desc_delta]</font>"
+				world << sound('sound/AI/aimalf.ogg') //AI got valid
+			if("fakebooms") //Micheal Bay is in the house !
```
Comment (wild-billy) [inline code/modules/admin/topic.dm]: Spelling, might be intended

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -2633,6 +2633,57 @@
...
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEE")
+				var/choice = input("How much high-budget explosions do you want ?", "Micheal Bay SFX Systems", 1) as num
+				if(choice > 0) //No negative or null explosion amounts here math genius
```
Comment (wild-billy) [inline code/modules/admin/topic.dm]: Always try to give feedback to the user when invalidating their input, just print them something like that comment

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -2633,6 +2633,57 @@
...
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEM")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Meteor Alert.")
+				command_alert("A meteor storm has been detected on collision course with the station. Seek shelter within the core of the station immediately.", "Meteor Alert")
+				world << sound('sound/AI/meteors.ogg')
+			if("fakecarps")
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","FAKEC")
+				message_admins("[key_name_admin(usr)] triggered a FAKE Carp Migration Alert.")
+				command_alert("Unknown biological entities have been detected near [station_name()], please stand-by.", "Lifesign Alert")
```
Comment (wild-billy) [inline code/modules/admin/topic.dm]: The above 3 are almost identical and could easily be merged into 1 proc with arguments: this is what @ririchiyo means

Comment (dylanstrategie) [inline code/modules/admin/topic.dm]: Aye, I'll look into it

**Comment by Duny-** [comment]
That's actually nice
I love it
Inb4 it breaks something

**Comment by ririchiyo** [comment]
I guess it's okay but not something that I wouldn't mind seeing reworked in the future.

**Pull request merged**